lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -374,7 +376,7 @@ private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
     private List<SignedJWT> getSignedJWTs(String userId) throws HttpResponseExceptionWithErrorBody {
         List<String> credentials =
                 userIdentityService.getUserIssuedCredentials(
-                        verifiableCredentialService.getVcStoreItems(userId));
+                        excludeTicfVC(verifiableCredentialService.getVcStoreItems(userId)));
```

Comment:
I wonder if we should actually move this check inside `userIdentityService.getUserIssuedCredentials`, and perhaps rename to `getIdentityCredentials`? - I don't think we'll ever want to use the TICF credential in any of our checks/identity operations.

That might actually mean we can remove the TICF evidence type from the GPG45 evaluator, because it'll never get that far :)

Reply:
Refactor done.